<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>20</y>
     <width>441</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>                                                 Sayı Analiz Etme Ekranı</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alt Sınır</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="range1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Üst Sınır</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="range2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="random">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rastgele Sayı Üret</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="tum">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tüm Çarpanlar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="asal">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Asal Çarpanlar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="listele">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Listele</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Çıkış</string>
    </property>
   </widget>
   <widget class="QLabel" name="uret">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Üretilen Sayı: </string>
    </property>
   </widget>
   <widget class="QListWidget" name="liste">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="liste_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
